lib/rich_text.rb

```diff
@@ -1,5 +1,45 @@
 # frozen_string_literal: true
 
+module Kramdown
```

Comment:
Injecting code into the `Kramdown` namespace is not ideal as we don't own that.

Reply:
https://kramdown.gettalong.org/rdoc/Kramdown/Parser/Base.html

> Implementing a new parser is rather easy: just derive a new class from this class and put it in the [Kramdown::Parser](https://kramdown.gettalong.org/rdoc/Kramdown/Parser.html) module – the latter is needed so that the auto-detection of the new parser works correctly.